data/src/main/java/com/teamagam/gimelgimel/data/message/repository/search/MessagesTextSearcherData.java

```diff
@@ -0,0 +1,37 @@
+package com.teamagam.gimelgimel.data.message.repository.search;
+
+import com.google.common.collect.Lists;
+import com.teamagam.gimelgimel.data.message.repository.cache.room.dao.SearchMessagesDao;
+import com.teamagam.gimelgimel.data.message.repository.cache.room.entities.ChatMessageEntity;
+import com.teamagam.gimelgimel.data.message.repository.cache.room.mappers.MessagesEntityMapper;
+import com.teamagam.gimelgimel.domain.messages.entity.ChatMessage;
+import com.teamagam.gimelgimel.domain.messages.search.MessagesTextSearcher;
+import java.util.List;
+import javax.inject.Inject;
+
+public class MessagesTextSearcherData implements MessagesTextSearcher {
```

Comment:
Again, this class name doesn't read right in English.
Rename it.

Reply:
done